setup.py

```diff
@@ -0,0 +1,7 @@
+from distutils.core import setup
+
+
+setup(name='day1',
+      version='0.1',
+      description='Production Code',
+      packages=['day1'])
```

Comment:
**EU** acho essa indentação meio feia... Prefiro algo assim:

```python
setup(
    name='day1',
    version='0.1',
    description='Production Code',
    packages=['day1']
)
```

E essa `description` pode ser algo mais voltado ao propósito do app? Tipo "Day 1 of Advent of Code 2015" ou "Help Santa to deliver presents in a large apartment building". Faz sentido? 🤔